.github/workflows/build.yaml

```
@@ -1,49 +1,43 @@
-name: Update devs api image
+name: Build, tag, and publish
 env:
   # This must be defined for the bash redirection
   GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
   # This must be defined for the bash redirection
   GOOGLE_SA_CERT: 'jade-dev-account.pem'
 on:
   workflow_dispatch: {}
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize  # https://github.community/t/what-is-a-pull-request-synchronize-event/14784/2
   push:
     branches:
       - develop
 jobs:
   bump_version:
-    runs-on: ubuntu-latest
-    outputs:
-      api_image_tag: ${{ steps.bumperstep.outputs.tag }}
     # Skip entire workflow if commit is authored by broadbot
     # broadbot is only used for automated commits, like version bumps
     # We don't want to trigger a version bump/deploy to dev for those
     if: ${{ !contains( github.event.sender.login, 'broadbot') }}
-    steps:
-      - name: Checkout Develop branch of jade-data-repo
-        uses: actions/checkout@v3
-        with:
-          ref: develop
-          token: ${{ secrets.BROADBOT_TOKEN }}
-      - name: "Bump the tag to a new version"
-        id: bumperstep
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'bumper'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-          version_file_path: build.gradle
-          version_variable_name: version
-          # Sets the author of the version bump commit to broadbot. This is used in our skip job logic.
-          GITHUB_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
+    uses: ./.github/workflows/tag.yml
```

Do you know if `./` is needed here? It seems redundant to me.
```suggestion
    uses: .github/workflows/tag.yml
```

---

Admittedly I haven't tried it that way, but GitHub's docs recommend the [clunkier version](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#calling-a-reusable-workflow), so I'm inclined to stick with that.

---

Thanks, I wasn't aware of this